posthog/admin/admins/redis_mutation_admin.py

```diff
@@ -0,0 +1,353 @@
+import json
+
+from django import forms
+from django.contrib import admin, messages
+from django.core.exceptions import ValidationError
+from django.http import HttpRequest
+from django.utils.translation import ngettext
+
+from posthog.models.redis import MutationFailedToSaveError, MutationInactiveError, RedisMutation
+from posthog.redis import get_client
+
+
+class RedisMutationForm(forms.ModelForm):
+    """Custom form for 'RedisMutationAdmin' to add custom styling and defaults."""
+
+    value = forms.CharField(empty_value=None)
```

Comment:
Notice we are taking this as a char field and later storing it JSON-encoded. This is done to support different types in one field, as we can encode numbers and strings (and even bools, eventually) as JSON.

Certainly a bit of a hacky shortcut.

Ideally we would have different forms for each type of mutation, and each form could narrow down the types and add as many specific fields as it needs. But, the PR was getting quite large, and from a usage perspective I didn't see much value from such a change. Moreover, it can always be something to iterate later on.